elephant/signal_processing.py

```diff
@@ -304,10 +304,15 @@ def butter(signal, highpass_freq=None, lowpass_freq=None, order=4,
     order : int
         Order of Butterworth filter. Default is 4.
     filter_function : string
-        Filtering function to be used. Either 'filtfilt'
-        (`scipy.signal.filtfilt()`) or 'lfilter' (`scipy.signal.lfilter()`). In
-        most applications 'filtfilt' should be used, because it doesn't bring
-        about phase shift due to filtering. Default is 'filtfilt'.
+        Filtering function to be used. Available filters:
+            * 'filtfilt': `scipy.signal.filtfilt()`;
+            * 'lfilter': `scipy.signal.lfilter()`;
+            * 'sosfiltfilt': `scipy.signal.sosfiltfilt()`.
+        In most applications 'filtfilt' should be used, because it doesn't
+        bring about phase shift due to filtering. For numerically stable
+        filtering, use 'sosfiltfilt' (see issue
+        https://github.com/NeuralEnsemble/elephant/issues/220).
```

Comment:
From what I understood, sos filtering decomposes a FIR filter or some order into a series of second order filters, so that this is useful in particular to generate higher filter orders (e.g., https://de.wikipedia.org/wiki/Filter_mit_endlicher_Impulsantwort#Kaskadierte_Form , sadly not translated in the English version of wikipedia). If this is true, maybe one could hint at this by saying, e.g., 
`For numerically stable filtering, in particular at higher filter orders, use sosfiltfilt...`

Reply:
Yeah, I wanted to mention higher order filtering but missed it at some point. I'll add the note.